src/tests/Equinor.ProCoSys.IPO.WebApi.Tests/Authorizations/ClaimsExtensionTests.cs

```diff
@@ -36,5 +36,127 @@ public void TryGetOid_ShouldReturnNull_WhenOidClaimNotExists()
             // Assert
             Assert.IsFalse(guid.HasValue);
         }
+
```

Comment:
Add yet a test: Add both ClaimTypes.GivenName + ClaimsExtensions.Name type of claims with different names, and proof that value from ClaimTypes.GivenName are used

Same for combo of ClaimTypes.SurName + ClaimsExtensions.Name